go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/mattn/go-isatty v0.0.20
 	github.com/moby/moby v27.1.1+incompatible
 	github.com/moby/term v0.5.0
-	github.com/ninech/apis v0.0.0-20241014111010-322e555390a0
+	github.com/ninech/apis v0.0.0-20241022115306-047cdfbe076b
```

Comment:
these changes depend on unmerged API changes so this will need to be updated again once the API is merged.